modules/nw-ingress-creating-a-route-via-an-ingress.adoc

```diff
@@ -47,6 +48,10 @@ as `Ingress` has no field for this. The accepted values are `edge`, `passthrough
 other values are silently ignored. When the annotation value is unset, `edge` is the default route. The TLS certificate details must be defined in the template file to implement the default edge route.
 
 .. If you specify the `passthrough` value in the `route.openshift.io/termination` annotation, set `path` to `''` and `pathType` to `ImplementationSpecific` in the spec:
+<2> The `route.openshift.io/destination-ca-certificate-secret` can be used on an Ingress Object to define a route with a custom certificate (CA). The annotation references a kubernetes secret, `secret-ca-cert` that is inserted into the generated route
+
+
+.. To specify an Ingress route with a CA, you must have a certificate/key pair in PEM-encoded files, where the certificate is valid for the route host. You may have a separate CA certificate in a PEM-encoded file that completes the certificate chain. You must have a separate destination CA certificate in a PEM-encoded file. You must have a service that you want to expose.
```

Comment:
I think we should reword this section. I just realized we aren't really talking about the secret and also I'm trying to figure out how much of this actually applies (I'm not a cert expert), but I think it might be better to link to the existing secured routing page. Maybe we can do that here. Here is my suggestion:

```suggestion
.. To specify an route object with a destination CA from an ingress object, you must create a `kubernetes.io/tls` or `Opaque` type secret with a certificate in PEM-encoded format in the `data.tls.crt` specifier of the secret. Refer to openshift-enterprise/latest/networking/routes/secured-routes.html (TODO fancy link this) for more information on secured routing and certificates.
```

Reply:
Okay, maybe @melvinjoseph86 has an opinion about this.